pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -2404,11 +2404,19 @@ func buildListener(port api.ServicePort, annotations map[string]string, sslPorts
 	return listener, nil
 }
 
+func nodeNames(nodes []*api.Node) sets.String {
```

Comment:
I can see your point about it likely being a temporary artifact as providers refactor.  Sounds good.